tendrl/commons/tests/test_init.py

```diff
@@ -461,6 +465,7 @@ def test_setup_common_objects(monkeypatch):
         tendrlNS.current_ns.objects["Config"] = obj_cls[1]
     with patch.object(etcd, "Client", return_value=etcd.Client()) as client:
         tendrlNS.current_ns.objects.pop("NodeContext")
+        NS.tendrl.objects.TendrlContext.load = MagicMock()
```

Comment:
Should we use MagicMock ? Most of the places we have been using Mock/patch. Asking just consistency.

Reply:
using mock/patch we can mock the function of a class which is existing one, but here I have created a mock function for a class in the name load. 

Because TendrlContext class does not have load function, only BaseObject class have load function, when we create an object for TendrlContext then only base class (Baseobject) properties should be accessible by child class Obj. But here I want to create one mock function for class TendrlContext before object creation. So child class method overrides base class.

Reply:
ack